pallets/msa/src/lib.rs

```diff
@@ -579,6 +579,11 @@ pub mod pallet {
 
 		/// Retire a MSA
 		///
+		/// When a user wants to disassociate themselves from Frequency, they can retire their MSA for free provided that:
```

Comment:
Perhaps, we can extend the description to say that retiring your MSA does not altogether remove all associated data. For example, messages related to an MSA are not removed.

Reply:
The intent is for it to remove it but we are not doing that currently.  I will update the description to explain what it is doing currently.

Reply:
/// This does not currently remove any messages related to the MSA.